Precompile error

After upgrading my packages in general and Revise specifically to 2.1.2 I'm getting 
```julia
julia> using Revise
[ Info: Precompiling Revise [295af30f-e4ad-537b-8983-00126c2a3abe]
ERROR: LoadError: AssertionError: precompile(getfield(Revise, m.name), (Bool, typeof(methods_by_execution!), Function, CodeTrackingMethodInfo, Dict{Module, Vector{Expr}}, Module, Expr))
Stacktrace:
 [1] _precompile_() at /Users/andreasnoack/.julia/packages/Revise/RW8TD/src/precompile.jl:16
 [2] top-level scope at none:0
 [3] include at ./boot.jl:326 [inlined]
 [4] include_relative(::Module, ::String) at ./loading.jl:1038
 [5] include(::Module, ::String) at ./sysimg.jl:29
 [6] top-level scope at none:2
 [7] eval at ./boot.jl:328 [inlined]
 [8] eval(::Expr) at ./client.jl:404
 [9] top-level scope at ./none:3
in expression starting at /Users/andreasnoack/.julia/packages/Revise/RW8TD/src/Revise.jl:1093
ERROR: Failed to precompile Revise [295af30f-e4ad-537b-8983-00126c2a3abe] to /Users/andreasnoack/.julia/compiled/v1.1/Revise/M1Qoh.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1197
 [3] _require(::Base.PkgId) at ./loading.jl:960
 [4] require(::Base.PkgId) at ./loading.jl:858
 [5] require(::Module, ::Symbol) at ./loading.jl:853
```
when running in the `v1.1` environment. It succeeds when I try installing the version in clean environment.

Any ideas what is going on and how I can debug? Already to delete `compiled`. Also, downgrading to `1.1.0` also fixed the issue. couldn't try intermediate versions because of `JuliaInterpreter`.

--- a/Project.toml
+++ b/Project.toml
@@ -18,7 +18,7 @@ Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 [compat]
 CodeTracking = "~0.5.1"
 JuliaInterpreter = "0.2, 0.3, 0.4, 0.5"
-LoweredCodeUtils = "~0.3.1"
+LoweredCodeUtils = "0.3.2"
 
 [extras]
 EponymTuples = "97e2ac4a-e175-5f49-beb1-4d6866a6cdc3"

--- a/Project.toml
+++ b/Project.toml
@@ -1,6 +1,6 @@
 name = "Revise"
 uuid = "295af30f-e4ad-537b-8983-00126c2a3abe"
-version = "2.1.2"
+version = "2.1.3"
 
 [deps]
 CodeTracking = "da1fd8a2-8d9e-5ec2-8556-3022fb5608a2"

